cogstat/cogstat_stat.py

```diff
@@ -553,6 +555,13 @@ def variable_pair_regression_coefficients(slope, intercept, std_err, intercept_s
         regression_coefficients = '<cs_h3>' + _('Regression coefficients') + '</cs_h3>'
         pdf_result = pd.DataFrame(columns=[_('Point estimation'), _('95% confidence interval')])
 
+        if not normality:
```

Comment:
We should also consider what should happen if normality couldn't be calculated (it is None).

Reply:
None case with warning added.